docs/UserGuide.md

```diff
@@ -502,35 +495,36 @@ the command line based on the indexes labelled in the figure.
 
 Format: `undone INDEX`
 
-You can unmark assignments that you have previously marked as done by specifying the assignment `INDEX` as shown in your
- list.
+If you realised that you've an uncompleted assignment marked as done, you can mark the assignment as not done.
+    
+Similar to done, you can mark an assignment as not done by specifying the assignment's `INDEX` shown in your list.
 
-You can unmark `one` done assignment at a time. Here is an example with steps for you to follow:
+Here is an example with steps for you to follow:
 
-1) If you have **accidentally** marked the assignment with the name "Essay" as done,
-you can simply enter `undone 2` into the command line based on the index labelled in the figure.
+1) If you realised that the "Essay" assignment has been marked as done accidentally,
+you can simply enter `undone 2` into the Command Box. `2` refers to the `INDEX` of the assignment as shown in the figure below.
```

Comment:
Maybe phrasing it as "If you realised that the assignment has been mistakenly marked as done" would sound more natural? Currently, it sounds a little Singlish to me XD

Reply:
okay thanks for the suggestions! :)